backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/employee/EmployeeInterface.java

```diff
@@ -0,0 +1,9 @@
+package de.neuefische.ffmjava221.teamprojekt.backend.employee;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface EmployeeInterface extends MongoRepository<Employee, String> {
```

Comment:
rename the employeeInterface to something more specific
for example : EmployeeDB, EmployeeRepo, EmployeeRepository